GCRCatalogs/SCHEMA.md

```diff
@@ -97,7 +97,7 @@ The schema for DC2 Coadd Catalogs follow the following rules:
 - For quantities that are defined in [LSST DPDD](https://lse-163.lsst.io/dpdd.pdf), we follow DPDD's naming scheme.
 - For quantities that are defined the above "Schema for Extragalatic Catalogs", we follow Extragalatic Catalogs' naming scheme.
 - For quantities that are defined in both, we provide aliases so both naming schemes would work.
-- For quantities that are defined in neither, we define them below:
+- For quantities that are defined in neither, we define them below. :
```

Comment:
This statement is confusing and not entirely accurate.  We should list all of the things we define here and state whether or not they're from the DPDD.

Reply:
this one addressed.